hclgen/hcl_gen.go

```diff
@@ -0,0 +1,659 @@
+package hclgen
+
+import (
+	"embed"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+
+	embedJson "github.com/hashicorp/waypoint/embedJson"
```

Comment:
Since this import is only used in this file, I don't think we need to turn it into a package, especially since we also import "embed" on line 4.